src/app/mail/mail-sidebar/compose-mail-dialog/compose-mail-dialog.component.ts

```diff
@@ -60,14 +60,9 @@ export class ComposeMailDialogComponent implements OnInit, AfterViewInit {
 
   onClose() {
     if (this.composeMail.hasData()) {
-      this.confirmModalRef = this.modalService.open(this.confirmDiscardModal, {
```

Comment:
delete the html code against this modal as well : `confirmDiscardModal `

Reply:
Okay sure.